styles.css

```diff
@@ -0,0 +1,1115 @@
+}
+
+.con-title {
+  color: #fcfcfc;
+  justify-content: center;
+  font-weight: bold;
+  line-height: 40px;
+  font-size: 40px;
+  font-family: Poppins, sans-serif;
+  margin: 150px auto 12px;
+}
+
+.con-paragraph {
+  font-family: Poppins, sans-serif;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  justify-content: center;
+  text-align: center;
+  font-weight: normal;
+  margin: 0 auto;
+  padding: 10px 50px;
+}
+
+.con-form {
+  display: flex;
+  flex-direction: column;
+  margin: 30px;
+  gap: 20px;
+}
+
+.form-text {
+  border: 1px solid white;
+  border-radius: 8px;
+  height: 48px;
+  padding: 15px;
+  font-family: Poppins, sans-serif;
+}
+
+.error-message {
+  color: red !important;
+  margin-top: 0.2rem;
+  font-size: 14px;
+  text-transform: uppercase;
+}
+
+.con-but {
+  justify-content: start;
+  width: 140px;
+  height: 48px;
+  padding: 10px;
+  border: 1px solid white;
+  border-radius: 8px;
+  font-family: Poppins, sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  color: #2e63e7;
+}
+
+.message {
+  height: 170px;
+  border-radius: 8px;
+}
+
+.skills-section {
+  position: relative;
+  bottom: 30%;
+}
+
+.technologies-langu li {
+  background-color: #ebebff;
+  padding: 10px;
+  border-radius: 10px;
+  font-weight: 500;
+  font-family: poppins, sans-serif;
+  justify-content: space-between;
+  margin-right: 20px;
+  display: inline-block;
+}
+
+.menu-desktop {
+  display: none;
+}
+
+.desktop-image {
+  display: none;
+}
+
+.desktop_show,
+.optiond2_full,
+.instruction-show,
+.dcanopy2_face,
+.dcanopy2_facebook,
+.desktop-image {
+  display: none;
+}
+
+@media (min-width: 992px) {
+  .sidebar {
+    display: none;
+  }
+
+  .lang {
+    padding-top: 40px;
+  }
+
+  .first-section {
+    background-image: url("images/header-desktop.png");
+    height: 100%;
+  }
+
+  .menu-desktop {
+    display: flex;
+    margin-left: auto;
+    margin-right: 120px;
+    visibility: visible;
+  }
+
+  .logo {
+    display: inline-block;
+    margin-left: 155px;
+    padding-left: 0;
+  }
+
+  .menu-desktop ul {
+    display: flex;
+  }
+
+  .menu-desktop li {
+    list-style: none;
+    padding: 12px;
+    text-decoration: none;
+  }
+
+  .menu-desktop li a {
+    text-decoration: none;
+    color: #344563;
+    font-size: 15px;
+    font-weight: 500;
+    font-style: normal;
+    background-color: #fff;
+  }
+
+  .menu-desktop li a:hover {
+    color: #4472e7;
+  }
+
+  .contact-form {
+    background-image: url("images/contact-desktop.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+    width: 100vm;
+    place-self: center;
+  }
+
+  .language {
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-start;
+    text-align: center;
+    width: 8em;
+  }
+
+  .langlist {
+    display: flex;
+  }
+
+  .lang-text {
+    text-align: center;
+  }
+
+  .about-grid {
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    padding-bottom: 100px;
+  }
```

Comment:
- [ ] great job using flex to layout this content, however, the **spacing(padding, margin)** is different than the one specified on the design, kindly fix that.

### Your website
![Screen Shot 2022-09-03 at 9 58 40 AM](https://user-images.githubusercontent.com/59416136/188259847-bce2d042-34a2-48e0-8f40-711312c8d542.png)

### Figma's design
![Screen Shot 2022-09-03 at 10 00 57 AM](https://user-images.githubusercontent.com/59416136/188259864-3ee89304-b424-4e1e-b307-b9b5136f3f88.png)

Reply:
thanks I have made the corrections